airbyte-cron/src/main/java/io/airbyte/cron/selfhealing/Temporal.java

```diff
@@ -4,19 +4,27 @@
 
 package io.airbyte.cron.selfhealing;
 
+import io.airbyte.workers.temporal.TemporalClient;
 import io.micronaut.scheduling.annotation.Scheduled;
+import io.temporal.api.enums.v1.WorkflowExecutionStatus;
+import javax.inject.Named;
 import javax.inject.Singleton;
 import lombok.extern.slf4j.Slf4j;
 
 @Singleton
 @Slf4j
 public class Temporal {
 
-  public Temporal() {
+  private final TemporalClient temporalClient;
+
+  public Temporal(@Named("temporalClient") final TemporalClient temporalClient) {
     log.info("Creating temporal self-healing");
```

Comment:
Can this info call be removed (or changed to debug)?

Reply:
Changed to debug